apps/public-docsite-v9/package.json

```diff
@@ -27,10 +27,8 @@
     "@fluentui/react": "^8.105.4",
     "@fluentui/react-northstar": "^0.66.0",
     "@fluentui/react-icons-northstar": "^0.66.0",
-    "@fluentui/storybook": "^1.0.0",
     "@fluentui/react-components": "^9.13.0",
     "@fluentui/react-storybook-addon": "9.0.0-rc.1",
-    "@fluentui/react-theme": "^9.1.5",
```

Comment:
this was here probably because legacy reasons to mitigate need of build whole react-components suite. we now have improved build speed and DX thus can be removed and imports can be normalized within code base

Reply:
Agree that this was likely legacy as the v8 docs needed the theme's referenced for the theme dropdown picker.